index.bs

```diff
@@ -246,7 +246,7 @@ The [=lock-concept/waiting promise=] is not named in the above code, but is stil
 Further note that if the callback is not `async` and returns a non-promise, the return value is wrapped in a promise that is immediately resolved; the lock will be released in an upcoming microtask, and the [=lock-concept/released promise=] will also resolve in a subsequent microtask.
 </aside>
 
-Each origin has a <dfn>held lock set</dfn> which is a [=/set=] of [=lock-concept|locks=].
+Each [=/lock manager=] has a <dfn for="lock manager">held lock set</dfn> which is a [=/set=] of [=lock-concept|locks=].
```

Comment:
Forgot to say this, should these "each lock manager has ..." statement be merged to the definition of lock manager?